django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -236,11 +241,12 @@ Requires jQuery, core.js, and SelectBox.js.
     };
 
     window.addEventListener('load', function(e) {
-        $('select.selectfilter, select.selectfilterstacked').each(function() {
-            var $el = $(this),
-                data = $el.data();
-            SelectFilter.init($el.attr('id'), data.fieldName, parseInt(data.isStacked, 10));
-        });
+        var elements = document.querySelectorAll('select.selectfilter, select.selectfilterstacked');
+        for (var i = 0; i < elements.length; i++) {
```

Comment:
`.forEach()` wasn't used here due to lack of support in the PhantomJS test runner.

Reply:
Fixed by upgrading grunt-contrib-qunit to use puppeteer instead of PhantomJS.